src/data/talentsData.ts

```diff
@@ -2,113 +2,238 @@ import { CardTalent } from '../interfaces/Cards/CardTalent'
+        url: 'https://twitter.com/Anthares101'
       }
     ]
   },
   {
+    trans_key: 'antonio_romero',
     imageUrl: '/img/talents/placeholder.png',
-    name: 'Miguel Gavilán',
-    description: 'Magía y hechicería',
+    name: 'Antonio Romero',
+    description: 'Antonio is a software developer, forerunner of good vibes, who enjoys programming, challenges, teamwork, coffee and music. Always willing to continue learning. The greater the effort, the greater the glory!.',
     socialNetworks: [
       {
-        name: 'Email',
-        url: ''
+        name: 'Linkedin',
+        url: 'www.linkedin.com/in/antonio-romero-cañete-a8488531'
       }
     ]
   },
   {
+    trans_key: 'nieves_borrero',
     imageUrl: '/img/talents/placeholder.png',
-    name: 'Javier Martín',
-    description: 'Magía y hechicería',
+    name: 'Nieves Borrero',
+    description: 'Nieves Borrero is a Software Developer who enjoys programming and converting code into a piece of art that improves other people’s lives that’s why she also is dedicated to colaborate as Organizer in Google Developers Group and Women TechMackers.',
     socialNetworks: [
       {
-        name: 'Email',
-        url: ''
+        name: 'Linkedin',
+        url: 'https://www.linkedin.com/in/nievesborrero/'
+      },
+      {
+        name: 'Twitter',
+        url: 'https://twitter.com/nievesborrero'
       }
     ]
   },
   {
+    trans_key: 'victoria_perez',
     imageUrl: '/img/talents/placeholder.png',
-    name: 'Daniel Ranchal',
-    description: 'Magía y hechicería',
+    name: 'Victoria Pérez',
+    description: 'description',
     socialNetworks: [
       {
-        name: 'Email',
-        url: ''
+        name: 'Linkedin',
+        url: 'https://www.linkedin.com/in/victoriapc/'
       }
     ]
   },
   {
+    trans_key: 'marisa_ruiz',
+    imageUrl: '/img/talents/placeholder.png',
+    name: 'Marisa Ruiz',
+    description: 'description',
+    socialNetworks: [
+    ]
+  },
+  {
+    trans_key: 'joaquin_lopez',
     imageUrl: '/img/talents/placeholder.png',
-    name: 'Nieves borrego',
-    description: 'Magía y hechicería',
+    name: 'Joaquín López',
+    description: 'description',
+    socialNetworks: [
+    ]
+  },
+  {
+    trans_key: 'toni_parn',
+    imageUrl: '/img/talents/placeholder.png',
+    name: 'Toni Pärn',
+    description: 'I am just here to help :).',
     socialNetworks: [
       {
-        name: 'Email',
-        url: ''
+        name: 'Linkedin',
+        url: 'https://www.linkedin.com/in/toni-p-13a08985/'
       }
     ]
   },
   {
+    trans_key: 'miriam_flores',
     imageUrl: '/img/talents/placeholder.png',
-    name: 'Victoria',
-    description: 'Magía y hechicería',
+    name: 'Miriam Flores',
+    description: 'She’s a versatile and dynamic person who seeks to excel in everything she does. She loves learning about completely different disciplines and developing herself in new areas because she’s one of those who think that all knowledge is good.',
```

Comment:
```suggestion
    description: 'She\'s a versatile and dynamic person who seeks to excel in everything she does. She loves learning about completely different disciplines and developing herself in new areas because she’s one of those who think that all knowledge is good.',
```